data-prepper-plugins/s3-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/s3/grouping/S3GroupIdentifierFactory.java

```diff
@@ -70,6 +70,6 @@ public S3GroupIdentifier getS3GroupIdentifierForEvent(final Event event) {
         }
 
 
-        return new S3GroupIdentifier(groupIdentificationHash, fullObjectKey, s3SinkConfig.getPredefinedObjectMetadata(), fullBucketName);
+        return new S3GroupIdentifier(groupIdentificationHash, fullObjectKey, s3SinkConfig.getObjectMetadataConfig(), fullBucketName);
```

Comment:
With this change, this `predefinedObjectMetadata` option is no longer used anywhere. So even if users config this `predefined_object_metadata` option, it will not be used. Is this expected?

Reply:
SInce the option `predefined_object_metadata` is already released in DataPrepper 2.9, David, did not want to just rename it. So, we will deprecate `predefined_object_metadata` in future.